docs/model_developers/modules.rst

```diff
@@ -347,10 +347,26 @@ Add the following content to the file:
   login <username>
   password <password>
 
-For more information see the doc about`pip authentication <https://pip.pypa.io/en/stable/topics/authentication/>`_.
+For more information see the doc about `pip authentication <https://pip.pypa.io/en/stable/topics/authentication/>`_.
 
 You will also need to specify the url of the repository in the ``project.yml`` file of your project (See: :ref:`specify_location_pip`).
 
 By following the previous steps, the Inmanta server will be able to install modules from a private Python package repository.
 
 
+Inter-module dependencies
+#########################
+
+The plugins code of a module A can have a dependency on the plugins code of another V2 module B. When doing this,
+care should be taken that the Python module(s) you depend on, do not define any resources or providers. Otherwise the
```

Comment:
```suggestion
care should be taken that the module(s) you depend on, do not define any resources or providers. Otherwise the
```

I think the "Python" qualifier could be more confusing than helpful. It may seem to imply that it's the specific python module (file) that you depend on, while in practice none of the Python modules in that inmanta module may have resources.